docs/decisions/infra/2023-05-25-provision-database-users-with-serverless-function.md

```diff
@@ -1,25 +1,25 @@
 # Provision database users with serverless function
 
-* Status: proposed
-* Deciders: @lorenyu @kyeah @shawnvanderjagt @rocketnova
-* Date: 2023-05-25
+- Status: proposed
```

Comment:
Several ADRs list their status as "proposed" still rather than "accepted." Out of curiosity, is these accurate @lorenyu?

Reply:
let's change to accepted thanks for flagging